.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This is intentionally checked in and helps vsocde know where to find our tests. Unrelated to images